mozecode/dose
# Gm seeds

I made the following changes:
- modify user-form to prevent autofill on password, change format to use placeholders instead of labels
- add seed json files
- add user seed json file
- add sequelize seeder files
- fix relations for phone in seeder json file and seed database with fake data
- change passwords for fake users to valid hash and test login

To test: 
- Re-roll the database with ```sequelize db:migrate:undo:all``` , ```sequelize db:migrate```, and ```sequelize db:seed:all```.
- Go into the datbase and do select statements for all the tables to see the seeded fake data.  
- Run ```npm start``` in your terminal and go to localhost:4000.  Try signing in as one of the fake users.  You should get a welcome message for that user.
